### ISSUE TITLE
plugin_metadata_checker throws error though defaults do not exists as mandatory wants

### ISSUE THREAD

**Issue by bmxp** [smarthomeNG/smarthome#352]
```bash
python3 tools/plugin_metadata_checker.py -c join

plugin_metadata_checker.py v1.7.1 - Checks the care status of plugin metadata

Check metadata of smart plugin 'join'

ERROR:   parameter 'device_id': mandatory and default cannot be used toget
         her

         If mandatory and a default value are specified togeather, mandato
         ry has no effect, since a value for the parameter is already spec
         ified (the default value).

ERROR:   parameter 'api_key': mandatory and default cannot be used togethe
         r

         If mandatory and a default value are specified togeather, mandato
         ry has no effect, since a value for the parameter is already spec
         ified (the default value).

2 errors, 0 warnings and 0 hints
```
definition of the ``plugin.yaml``:
```yaml
#...
parameters:
    # Definition of parameters to be configured in etc/plugin.yaml
    device_id:
        type: str
        mandatory: True
        description:
            de: 'Join Device ID'
            en: 'Your own personal Join device id.'

    api_key:
        type: str
        mandatory: True
        description:
            de: 'Persönlicher Join API key'
            en: 'Your own personal Join api key.'
```

No default given here ...

**Comment by msinn** [comment]
Fixed, v1.7.2 of the tool pushed to develop

**Comment by bmxp** [comment]
Confirmed. No further problems found, I will close this :-) Thank you!
